proto/ratis-grpc.proto

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
A bigger question, not a blocking comment, but this should go to Ratis repo and not to Ozone go client repo? and we should take dependency on Ratis code.

For initial discussion on building a Ozone native client. Such idea was discussed as well, where the ratis code should go into ratis code ?

Reply:
Thanks the suggestinos.

I don't see such a recommendation in the thread, but I think it's a good idea.

But in the current phase of the project I would prefer to keep the Ratis related code here as we don't need full, Ratis client (implementing all the Ratis calls)  the only thing what we need to do is wrap the request in the Ratis envelope.

After we have a working demo: I agree. We can move out the Ratis related code, create a full-featured Ratis client and switch to use it. Until that I would prefer to keep here.

Reply:
ok, great thought. Lets create a follow up jira to that this will get done hopefully eventually ? Lets create a sub task under the master jira for go client